public/js/src/client.js

```diff
@@ -566,4 +591,14 @@ $(document).ready(function(){
 
 	// Begins the show!
 	setup();
-});
\ No newline at end of file
+});
+
+/* 
```

Comment:
This note will probably be the most interesting to you.  It leads the way to conversations about Browserify, HTTP2, and the future of front-end code.